bigchaindb/upsert_validator/validator_election.py

```diff
@@ -145,3 +150,79 @@ def create(cls, tx_signers, recipients, metadata=None, asset=None):
     @classmethod
     def transfer(cls, tx_signers, recipients, metadata=None, asset=None):
         raise NotImplementedError
+
+    @classmethod
+    def to_public_key(cls, election_id):
+        return base58.b58encode(bytes.fromhex(election_id))
+
+    @classmethod
+    def count_votes(cls, election_pk, txns, getter=getattr):
+        votes = 0
+        for txn in txns:
+            if getter(txn, 'operation') == 'VALIDATOR_ELECTION_VOTE':
+                for output in getter(txn, 'outputs'):
+                    # NOTE: We enforce that a valid vote to election id will have only
+                    # election_pk in the output public keys, including any other public key
+                    # along with election_pk will lead to vote being not considered valid.
+                    if len(getter(output, 'public_keys')) == 1 and [election_pk] == getter(output, 'public_keys'):
+                        votes = votes + int(getter(output, 'amount'))
+        return votes
+
+    def get_commited_votes(self, bigchain, election_pk=None):
+        if election_pk is None:
+            election_pk = self.to_public_key(self.id)
+        txns = list(backend.query.get_asset_tokens_for_public_keys(bigchain.connection,
+                                                                   self.id,
+                                                                   [election_pk],
+                                                                   'VALIDATOR_ELECTION_VOTE'))
+        return self.count_votes(election_pk, txns, dict.get)
+
+    @classmethod
+    def conclude(cls, bigchain, txn_id, current_votes=[], height=None):
```

Comment:
`conclude` sounds like an action, like you want to conclude the election right now, while this function is a check. I'd call it `is_concluded` or `has_concluded`.

I'm not sure about the semantic, the docstring says "Check if the given election", but there is no election explicitly passed to the function.

Reply:
renamed to `has_concluded` and updated the doc string.